json-endpoints/class.wpcom-json-api-site-settings-endpoint.php

```diff
@@ -1,5 +1,7 @@
 <?php
 
+include_once JETPACK__PLUGIN_DIR . 'functions.protect.php';
```

Comment:
Will this work both on wpcom and Jetpack?  Should those functions be in the root or in a module folder?